app/views/course/assessment/assessments/index.html.slim

```diff
@@ -2,9 +2,8 @@
   = new_button([current_course, :assessment]) if can?(:create, Course::Assessment.new(tab: @tab))
 
 = tabs do
-  - @category.tabs.each do |item|
-    = nav_to(format_inline_text(item.title),
-             course_assessments_path(current_course, category: @category, tab: item))
+  - assessment_tabs.each do |item|
```

Comment:
I was thinking more along the lines of this line just being

```ruby
= display_assessment_tabs
```

then in the helper

```ruby
return nil if @category.tabs == 1
tabs do
  @category.tabs.each...
end
```

The current changes doesn't seem idiomatic Ruby -- you've passed a block to tabs, and assessment_tabs is just an array (which is exactly the same as `@category.tabs`). Just place all the HTML logic in the helper.